src/mapper/map.rs

```diff
@@ -474,7 +474,7 @@ impl EventWriter {
             slot: source.header.header_body.slot,
             epoch: relative_epoch.map(|(epoch, _)| epoch),
             epoch_slot: relative_epoch.map(|(_, epoch_slot)| epoch_slot),
-            previous_hash: hex::encode(source.header.header_body.prev_hash),
+            previous_hash: source.header.header_body.prev_hash.map(hex::encode).unwrap_or_default(),
```

Comment:
I'm not sure what you want to do here. Did you want to make `previous_hash` in the BlockRecord be `Option<>`? I did this hacky solution just to unblock myself